src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+@import "../styles/variables.scss";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  justify-content: center;
+}
+
+.container {
+  padding-top: 150px;
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-padding;
+  gap: $box-gap;
+  width: calc(7 * #{$side} + 6 * #{$box-gap} + 2 * #{$calendar-padding});
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $side;
+    width: $side;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+
+    @for $days from 1 through 31 {
+      &:nth-child(#{$days}):before {
+        content: "#{$days}";
+      }
+    }
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
+    }
+  }
```

Comment:
Checklist point number 9. At the moment your transition work smoothly only when mouse income to the block